src/lando/ui/jinja2/stack/partials/uplift-section.html

```diff
@@ -60,3 +61,106 @@ <h1>Uplifts</h1>
         </div>
     </form>
 </div>
+{% endif %}
+
+{% if uplift_requests %}
```

Comment:
Is there a change that `uplift_requests` isn't declared here? If so, we could do with a `|default([])`

Reply:
No, it should always be defined when this template renders.

As a follow-up, I'd like to move all the uplift related pieces into some kind of helper object. Then we could pass that helper into the templating context and it could make the correct decisions about what to render.